persistence/atlas/src/main/atlas/spline-meta-model.json

```diff
@@ -0,0 +1,604 @@
+{
```

Comment:
Is it recognizable from the Atlas UI that the lineage was published by Spline?

Reply:
How compatible within Atlas is this with say NifiLineageReporter? I think this is its meta model: https://github.com/vakshorton/NifiAtlasLineageReporter/blob/master/AtlasTypes/nifiTypes.json

Reply:
> Is it recognizable from the Atlas UI that the lineage was published by Spline?

Explicitly no, but you can see all types defined in this meta-model in Atlas UI and those refer to Spark. IMHO, an explicit information about Spline doesn't give a user any additional value. WDYT?
>  How compatible within Atlas ...

see specification of ```FileEndpoint``, it's published as a built-in Atlas Type (hdfs_path), so if any other tool produce an entity with the same URI, the lineage graphs will join.